c2corg_ui/templates/outing/view.html

```diff
@@ -36,15 +36,20 @@
 
 <%block name="moduleConstantsValues">
   <%
-      geometry = outing['geometry']['geom_detail'] \
-          if outing['geometry']['geom_detail'] else outing['geometry']['geom']
+      if outing.get('geometry'):
+          geometry = outing['geometry']['geom_detail'] \
+              if outing['geometry']['geom_detail'] else outing['geometry']['geom']
+      else:
+          geometry = None
   %>
   module.value('mapFeatureCollection', {
     "type": "FeatureCollection",
     "properties": {},
     "features": [{
       "type": "Feature",
-      "geometry": ${geometry | n},
+      % if geometry:
```

Comment:
What's the point of adding a feature with no geometry?
If no geometry is available, the feature should be skipped from the featurecollection.